.gitignore

```diff
@@ -37,6 +38,7 @@ logs/
 tmp/
 VARS_PROD*.env
 pgpass*
+api/gnmatcher_name_strings_canonical.dump
```

Comment:
You actually staged and committed it at 60c033dc810085cb10fa9868af3b25139bc0837c

Moreover, there is https://github.com/GlobalNamesArchitecture/gnresolver/issues/58 that would make that ignore useless.